org.eclim.jdt/java/org/eclim/plugin/jdt/command/dependency/AddDependencyCommand.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (C) 2005 - 2016  Eric Van Dewoestine
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.eclim.plugin.jdt.command.dependency;
+
+import org.eclim.Services;
+import org.eclim.annotation.Command;
+import org.eclim.command.CommandException;
+import org.eclim.command.CommandException.ErrorType;
+import org.eclim.command.CommandLine;
+import org.eclim.command.Options;
+import org.eclim.logging.Logger;
+import org.eclim.plugin.core.command.AbstractCommand;
+import org.eclim.plugin.core.util.PathUtil;
+import org.eclim.plugin.core.util.PathUtilException;
+
+@Command(
+      name = "add_dependency",
```

Comment:
For consistency and to prevent collisions, all language specific commands have a common prefix. Since this is a jdt (java) command, this should have a `java_` prefix. For consistency with the existing classpath commands I'd call this `java_classpath_add_entry`.